docs/modules/ROOT/pages/exit.adoc

```diff
@@ -0,0 +1,21 @@
+= exit
+
+Exit command exits with the given code.
+
+Exit command can only be used in the interactive and scripting modes.
+
+If the given code is not in `[0,255]` range, it exists with code `255`.
```

Comment:
in the range `[0,255]`, it ??? with code

??? typo and should be exits ???

Reply:
https://github.com/hazelcast/clc-docs/pull/5/commits/156434bc9a0369bc6690f0c7071ef6942ee01478